data/styles/epub3-fonts.css

```diff
@@ -2,93 +2,77 @@
   font-family: "Noto Serif";
   font-style: normal;
   font-weight: normal;
-  src: url(../fonts/notoserif-regular-latin.ttf);
+  src: url("../fonts/notoserif-regular-latin.ttf");
```

Comment:
My understanding is that the quotes are not required when you use the round bracket enclosure. Is there a reason you are adding the double quotes?